content/en/docs/contributing/pr-checks.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Pull Request Checks
+description: Learn how to make your Pull Request successfully pass all checks
+weight: 40
+cSpell:ignore: REFCACHE
+---
+
+When you raise a
+[Pull Request](https://docs.github.com/en/get-started/learning-about-github/github-glossary#pull-request)
+with the
+[opentelemetry.io repository](https://github.com/open-telemetry/opentelemetry.io)
+a set of checks will be run. Those checks will verify, …
+
+- … if you have signed the [CLA](#easy-cla).
+- …if your commit can be deployed via [Netlify](#netlify-deployment)
+  successfully.
+- … if your changes are in line with some [style guidelines](#style-checks).
+
+{{% alert title="Note" color="primary" %}}
+
+If any of your PR checks fails, try to
+[fix content issues automatically](/docs/contributing/#fix-content-issues-automatically)
+first by running `npm run test-and-fix` locally.
+
+Additionally, you can comment `/fix:all` on your Pull Request. This will make
+the OpenTelemetry Bot run those commands on your behalf and update your PR. Make
+sure that you pull those changes locally.
```

Comment:
```suggestion
If any of the PR checks fails, try to
[fix content issues automatically](/docs/contributing/#fix-content-issues-automatically)
by running `npm run test-and-fix` on your machine.

Additionally, you can comment `/fix:all` on your Pull Request. This will make
the OpenTelemetry Bot run those commands on your behalf and update the PR. Make
sure that you pull those changes locally.
```